### DISCUSSION TITLE
Not linking to a phone number on README.md

### DISCUSSION THREAD

**Discussion by NahuelGenchi** [community/community#28811] [New to GitHub]
Hello, this is my first issue on Github.
Sorry if my explaining is not good, I'm trying to do my best.

I'm having a problem with a README.md file. I can't create a link to a phone number, it shows like plain text.

For example, I create the following code:

``
[+54 11 2222-2222](tel:+541122222222)
``
And this is the result: [+54 11 2222-2222](tel:+541122222222)

![github_issue](https://user-images.githubusercontent.com/65604505/183516263-16d842a2-b199-4e17-845a-9cae81bb7085.png)

It is supposed to create a link to the +541122222222 phone number, but it shows as plain text.

**Comment by radar-blip** [comment]
I think you have your brackets and paranthesis mixed up. [this](then this)

**Reply by NahuelGenchi** [reply]
The code is not mixed up:
![image](https://user-images.githubusercontent.com/65604505/183523998-f97363f4-6295-4e31-8f7a-6d4490a44e02.png)

**Reply by radar-blip** [reply]
Ok, try like this: 
```
[Phone](<a href="54 11 2222-2222">)
```
results:
[Phone](<a href="54 11 2222-2222">)

Note: You might have to edit format of your phone number.

**Reply by NahuelGenchi** [reply]
It doesn't work. It redirects to "/a href="54 11 2222-2222", which is an URL that doesn't exist.

**Reply by radar-blip** [reply]
Sorry, I guess I dont know how to link to a phone number. Maybe somebody else will chime in. 👍🏼

**Comment by theultramage** [comment]
Random passerby here, do you know if this soft of custom linking is even supported by github? Do you have a working example of it being used somewhere else?
I tried looking it up in the github docs, but the closest I got was [autolinks](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources) (paid only), and this doc on [github flavored markdown](https://github.github.com/gfm/#links). Didn't have time to fully dig through it.

**Reply by NahuelGenchi** [reply]
I don't know if this is supported by Github, but it should be. I can, for example, link a mail with the attribute mailto:

``[example@gmail.com](mailto:example@gmail.com)`` -> [example@gmail.com](mailto:example@gmail.com)
``[+54 11 2222-2222](tel:+541122222222)`` -> [+54 11 2222-2222](tel:+541122222222) (not working, it's not a link)

As you can see, if you click the example@gmail it redirects you to your mail default app to send an e-mail to the account specified.

So, if the mailto attribute works, why is the tel attribute not working?

**Reply by theultramage** [reply]
Github seems to have a whitelist for URI schemes, consisting of http, https and mailto. I tried ftp, irc, skype and some made-up schemes, and all of them cause the link to be converted to plain text. The autolinks section mentions that schemes have to be 'registered' (paid feature?). I found that you can write raw html in here, say `<a href="mailto:asdf">test</a>`, but the moment you put some other scheme in there, it turns into text. So there is some filter actively preventing the use of arbitrary schemes. Supporting tel: across github might require a feature request.
